tests/networks/nets/test_swin_unetr.py

```
@@ -101,6 +101,19 @@ def test_invalid_input_shape(self):
 
+    @skipUnless(has_einops, "Requires einops")
+    def test_flash_attention(self):
+        input_param = {"in_channels": 1, "out_channels": 2, "feature_size": 12, "spatial_dims": 3}
+        net_ref = SwinUNETR(use_flash_attention=False, **input_param)
+        net_flash = SwinUNETR(use_flash_attention=True, **input_param)
+        net_flash.load_state_dict(net_ref.state_dict())
+        x = torch.randn(1, 1, 64, 64, 64)
+        with eval_mode(net_ref, net_flash):
+            ref = net_ref.swinViT(x, net_ref.normalize)
+            out = net_flash.swinViT(x, net_flash.normalize)
+        for a, b in zip(ref, out):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)